pages/common/httprobe.md

```diff
@@ -0,0 +1,20 @@
+# httprobe
+
+> Take a list of domains and probe for working HTTP and HTTPS servers.
+> More information: <https://github.com/tomnomnom/httprobe>.
+
+- Probe a list of domains from a text file:
+
+`cat {{input_file}} > httprobe`
```

Comment:
This will send the output of the input file into a file called `httprobe` - it won't call `httprobe` at all.
```suggestion
`httprobe < {{path/to/input_file}}`
```

Reply:
Oops, good catch. That should be a pipe.